### PR TITLE
Changed docker image to Alpine

### PR DESCRIPTION
**Pull request by MichaelRobinson-BJSS**

### CODE REVIEW

**Review by PhysicsJon — DISMISSED** [review]

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
 #See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.
 
-FROM mcr.microsoft.com/dotnet/aspnet:5.0-buster-slim AS base
+FROM mcr.microsoft.com/dotnet/aspnet:5.0-alpine AS base
```
Comment (bjss-chris) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Dockerfile:3]: Does this work? Historically, alpine images have needed a couple of extra commands to work with SQL Server. Usually:

```Dockerfile
RUN apk update && apk add icu-libs
ENV DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=false
```

However, if it works then I guess MS may have updated the base image.

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Properties/launchSettings.json [hunk 2]
```diff
@@ -47,7 +47,19 @@
...
       "publishAllPorts": true,
-      "useSSL": true
+      "useSSL": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "ASPNETCORE_HOSTINGSTARTUPASSEMBLIES": "Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation",
+        "OPERATING_MODE": "Private",
+        "BC_DB_CONNECTION": "Server=localhost,1432;Database=GPITBuyingCatalogue;User=SA;password=Abc123Abc123;Integrated Security=false",
+        "CO_DB_CONNECTION": "Server=localhost,1432;Database=CatalogueOrdering;User=SA;password=Abc123Abc123;Integrated Security=false",
+        "APPINSIGHTS_INSTRUMENTATIONKEY": "ca1b863b-eb9a-4a81-8a23-182df28d1a0f",
```
Comment (bjss-chris) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Properties/launchSettings.json:57]: Should we be committing this to source?

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Dockerfile (hunk 1 above)
Comment (MichaelRobinson-BJSS) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Dockerfile:3]: Good spot, no it didn't work without the changes. Done

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Properties/launchSettings.json (hunk 2 above)
Comment (MichaelRobinson-BJSS) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Properties/launchSettings.json]: The AppInsights key shouldn't be in, but its obsolete anyway. I've removed it.
The connection strings (will ultimately just be one) don't reveal anything. Its just local test accounts

**Review by leerogers-bjss — APPROVED** [review]

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Properties/launchSettings.json (hunk 2 above)
Comment (bjss-chris) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Properties/launchSettings.json]: Yeah, it was the app insights key that I was referring to primarily. Still need to tidy up the connection strings but can do that later.

**Pull request merged**